src/App/App.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
```

Comment:
Consider separating third-party imports from the project's ones.
So first imports in a file should be third-party imports like "react".
Then your project common imports that appear in almost all the components - like for example "../../utils/your-util.js".
Then your component related stuff - like "./your-component-helper.js".
And then the css imports.

Every import group should be separated from other import groups with an empty new line.

Reply:
I've fixed my jsx files.Now it looks nice.
![image](https://user-images.githubusercontent.com/46630057/74604768-0ecea780-50d2-11ea-9d51-df2264ba3232.png)